homeassistant/components/conversation/trigger.py

```diff
@@ -49,14 +49,20 @@ async def async_attach_trigger(
     job = HassJob(action)
 
     @callback
-    async def call_action(sentence: str) -> str | None:
+    async def call_action(sentence: str, result: RecognizeResult) -> str | None:
         """Call action with right context."""
         trigger_input: dict[str, Any] = {  # Satisfy type checker
             **trigger_data,
             "platform": DOMAIN,
             "sentence": sentence,
         }
 
+        # Add wildcard values as extra trigger data
+        trigger_input["wildcards"] = {
```

Comment:
Initialize this key when dict is initialized

Reply:
fixed in [25e2b3a](https://github.com/home-assistant/core/pull/97236/commits/25e2b3a85223a7a4f133b283c482572ee462fa85)